processing/src/test/java/org/apache/druid/timeline/partition/OvershadowableManagerTest.java

```diff
@@ -165,6 +165,35 @@ public void testFindOvershadowedBy()
     );
   }
 
+  @Test
+  public void testHandleOutOfRangeFindOvershadowedBy()
+  {
+    final List<PartitionChunk<OvershadowableInteger>> expectedOvershadowedChunks = new ArrayList<>();
+    PartitionChunk<OvershadowableInteger> chunk = newNonRootChunk(32001, 32003, 1, 1);
+    manager.addChunk(chunk);
+    expectedOvershadowedChunks.add(chunk);
```

Comment:
Lets add a test case which is one more than `MAX_PARTITIONS_PER_TIMECHUNK` as well.